### ISSUE TITLE
line_length false positive?

### ISSUE THREAD

**Issue by Sega-Zero** [realm/SwiftLint#2175]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Bug Report

Having a false positive line_length warning. The comment text should not be taken into account.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ pbpaste | swiftlint lint --no-cache --use-stdin --enable-all-rules
<nopath>:3:10: warning: Closure Spacing Violation: Closure expressions should have a single space inside each brace. (closure_spacing)
<nopath>:4: warning: Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)
<nopath>:3: warning: Line Length Violation: Line should be 120 characters or less: currently 123 characters (line_length)
Done linting! Found 3 violations, 0 serious in 1 file.
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
```bash
$ swiftlint version
0.25.1
```
* Installation method used (Homebrew, CocoaPods, building from source, etc)?</br>Homebrew
* Paste your configuration file</br>No configuration, using defaults

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?</br>No
* Which Xcode version are you using (check `xcode-select -p`)?</br>Version 9.3 (9E145)
* Do you have a sample that shows the issue? Run `echo "[string here]" | swiftlint lint --no-cache --use-stdin --enable-all-rules`
  to quickly test if your example is really demonstrating the issue. If your example is more
  complex, you can use `swiftlint lint --path [file here] --no-cache --enable-all-rules`.

```swift
public convenience init(frame: CGRect, image: UIImage) {
    self.init(frame: frame)
    // This triggers a violation:
    ({self.image = image})() // Force Swift to call the property observer by calling the setter from a non-init context
}
```

**Comment by Nef10** [comment]
@Sega-Zero with the default configuration comments are taken into account for the `line_length` rule, but the rule has a configuration to change this:

```
line_length: 
  ignores_comments: true
```

**Comment by Sega-Zero** [comment]
That's a very strange default IMHO. Is there a reason for this rule to be enabled by default?

**Comment by marcelofabri** [comment]
That was one of the first implemented rules. Default rules are just a guideline, you should feel free to adapt to how your team works.
